lib/plugins/aws/package/compile/events/s3/index.test.js

```diff
@@ -19,21 +20,72 @@ describe('AwsCompileS3Events', () => {
 
   beforeEach(() => {
     addCustomResourceToServiceStub = sinon.stub().resolves();
-    const AwsCompileS3Events = proxyquire('./index', {
+    const AwsCompileS3EventsProxyquire = proxyquire('./index', {
       '../../../../customResources': {
         addCustomResourceToService: addCustomResourceToServiceStub,
       },
     });
     serverless = new Serverless();
     serverless.service.provider.compiledCloudFormationTemplate = { Resources: {} };
     serverless.setProvider('aws', new AwsProvider(serverless));
-    awsCompileS3Events = new AwsCompileS3Events(serverless);
+    awsCompileS3Events = new AwsCompileS3EventsProxyquire(serverless);
     awsCompileS3Events.serverless.service.service = 'new-service';
   });
 
   describe('#constructor()', () => {
     it('should set the provider variable to an instance of AwsProvider', () =>
       expect(awsCompileS3Events.provider).to.be.instanceof(AwsProvider));
+
+    describe('validation with extended schema', () => {
```

Comment:
I would not introduce any new tests here. As in this PR we just leave placeholders for validation, so I'd add real tests with PR that introduces complete schema (in same PR I would probably also refactor tests to `runServerless`, that'll make them real bulletproof)

Reply:
Thanks that's a big relief for me. Otherwise, it would require much more efforts to prepare tests for all the 15+ files